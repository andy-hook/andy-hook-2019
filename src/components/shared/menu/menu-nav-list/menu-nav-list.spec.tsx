import React from "react"

import MenuNavList from "./menu-nav-list"
import { shallowWithTheme } from "@test-utils"

describe("<MenuNavList />", () => {
  it("renders correctly", () => {
    const tree = shallowWithTheme(
      "light",
      <MenuNavList onClick={jest.fn} open={true} />
    )
    expect(tree).toMatchSnapshot()
  })
})
