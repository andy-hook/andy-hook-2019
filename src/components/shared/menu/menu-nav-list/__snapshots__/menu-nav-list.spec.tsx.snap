// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MenuNavList /> renders correctly 1`] = `ShallowWrapper {}`;
